tests/test_app_config.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+from django.test import TestCase
+
+from main import settings
+
+
+class TestConfig(TestCase):
+    def test_app_testing_config(self):
+        self.assertIsNotNone(settings.SECRET_KEY)
+        self.assertNotEqual(settings.SECRET_KEY, "")
+
+
+if __name__ == "__main__":
```

Comment:
this is not needed.
Don't import `unittest` and remove these last lines also because the test will run without this.